docs/source/faqs.md

```diff
@@ -76,6 +77,13 @@ We encourage TRE {ref}`Operators and Builders <infrastructure_roles>` to publicl
 
 Absolutely not. We know that TREs vary greatly in their design architecture, purposes for being built, the kinds of research they support and data they handle. We have tried to build a specification with as broadly useful a set of capabilities as possible, whilst acknowledging these different approaches. We won't have covered everything, and if you find SATRE valuable but think there is something we've missed, please consider {ref}`contributing <contributing>`. Additionally, the best practices in TRE provision may evolve over time as technologies and regulations change. We hope that the SATRE specification will be maintained in the future and accommodate these changes as appropriate.
 
+(other_tre_types)=
+
+## My TRE is designed for data that doesn't require this level of protection. Should I still follow SATRE?
+
+At the moment SATRE is a single specification that is relevant to many TREs.
```

Comment:
```suggestion
Yes. At the moment, the SATRE specification contains a set of capabilities marked as "Mandatory" which we believe are essential for a system to be defined as a TRE, as well as many "Recommended" and "Optional" capabilities.
Some of the non-mandatory capabilities will likely not be needed for TREs containing data that does not require all the possible protections, and there may well be tradeoffs to be made in terms of accessibility vs security that depend on the data the TRE holds.
```

I've tried to set up the next part about archetypes/tiers with a bit more context